spec.bs

```diff
@@ -4075,12 +4144,16 @@ from querying the server during an auction.
 <div algorithm>
   To <dfn>update k-anonymity cache for interest group</dfn> given an [=interest group=] |ig|:
     1. [=list/For each=] |igAd| of |ig|'s [=interest group/ads=]:
-      1. Compute the |adHashCode| following [=compute the key hash of ad=] for |ig| and |igAd|.
+      1. Let |adHashCode| be the result of running [=compute the key hash of ad=] given |ig| and |igAd|.
       1. Run [=update k-anonymity cache for key=] on |adHashCode|.
-      1. Compute the |adReportingHashCode| following [=compute the key hash of reporting ID=].
+      1. Let |adReportingHashCode| be the result of [=computing the key hash of reporting ID=] given |ig|, |igAd|, and null.
       1. Run [=update k-anonymity cache for key=] on |adReportingHashCode|.
+      1. If |igAd|'s [=interest group ad/selectable buyer and seller reporting IDs=] is not null:
+        1. [=list/For each=] |selectableReportingId| in |igAd|'s [=interest group ad/selectable buyer and seller reporting IDs=]:
+          1. Set |adReportingHashCode| be the result of [=computing the key hash of reporting ID=] given |ig|, |igAd|, and |selectableReportingId|.

```

Comment:
just to double check, only "selectableReportingId" is used in |adReportingHashCode|, not the other reporting ids, right?

Reply:
oh I think I looked at a wrong algorithm (a few k-anon algorithms have similar names 😂 ). Nevermind.